test/conformance/kuttl/policy-validation/cluster-policy/policy-exceptions-disabled/README.md

```diff
@@ -0,0 +1,7 @@
+## Description
+
+This test is attempting to create a resource with the label "app: my-test-app", which would typically violate the policy defined. However, there is a policy exception defined for resources with the same label, which should bypass the policy. However, the policy exception for the label "app: my-test-app" has been explicitly disabled, meaning that the resource should still fail to be created despite being a matching exception.
```

Comment:
Last sentence:

"Since the Policy Exception feature has not been enabled, the resource will be blocked by the policy instead of being allowed."